### ISSUE TITLE
A query with $match and/or $project gives an error

### ISSUE THREAD

**Issue by vasilev-alex** [Microsoft/vscode-cosmosdb#691]
Repro steps:
```
db.users.aggregate([
    { $match: {_id: ObjectId("5b23d2ba92b52cf794bdeb9c")}},
    { $project: {
        scores: {$filter: {
            input: '$scores',
            as: 'score',
            cond: {$gt: ['$$score', 3]}
        }}
    }}
])
```

Action: cosmosDB.executeAllMongoCommands
Error type: Error
Error Message: Error near line 1, column 6, text '$match'. Please check syntax.

Version: 0.7.2
OS: macOS
MongoDB shell version v3.6.5

The collection structure is somewhat like this
`{
	"_id" : ObjectId("5b23d2ba92b52cf794bdeb9c"),
	"scores" : [1, 2, 3, 4]
}`

Executing the same query in the shell gives no error

**Comment by PrashanthCorp** [comment]
Hi Alex, 
We currently don't support non-quoted or single-quoted strings. See #466 #467. 
I guess that could be one of the issues you faced. 

Can you try placing strings such as `input`,`cond`,etc. and `$project`,`$match`,etc. in double quotes? Please let us know if that solves the problem. I'm closing this as I suspect a duplicate. Feel free to reopen if the problem persists. 

FWIW, we currently are working on getting single quotes working fine, see #685 .

**Comment by PrashanthCorp** [comment]
Hi Alex, the merged fix was published today (version 0.8.0). Hope this fixes your issue. :-)
Let us know how the fix holds up with your use-case.
